OutOfSchool/OutOfSchool.RazorTemplatesData/Services/EmailTemplates.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace OutOfSchool.RazorTemplatesData.Services
+{
+    public static class EmailTemplates
+    {
+        /// <summary>
+        /// Bass path to the html email templates
```

Comment:
I hope in `/// Bass path` you mean `Base path`. Please correct here and in `MainPathPlainText`.